# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2022-2025, PyMC Labs
# This file is distributed under the same license as the pymc-marketing
# package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2025.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: pymc-marketing local\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-10-07 18:10+0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language: es\n"
"Language-Team: es <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.17.0\n"

#: ../source/api/generated/classmethods/pymc_marketing.mmm.budget_optimizer.BudgetOptimizer.model_rebuild.rst:2
#: cce730c15d5f4361a335af44bcb41832
msgid "BudgetOptimizer.model\\_rebuild"
msgstr "BudgetOptimizer.model\\_rebuild"

#: b966573b41c04f16996c8412cd6c8ed1 of pydantic.main.BaseModel.model_rebuild:2
msgid "Try to rebuild the pydantic-core schema for the model."
msgstr "Intente reconstruir el esquema de pydantic-core para el modelo."

#: d79dc50c70fe4ef49074412f57ed2fd4 of pydantic.main.BaseModel.model_rebuild:4
msgid ""
"This may be necessary when one of the annotations is a ForwardRef which "
"could not be resolved during the initial attempt to build the schema, and "
"automatic rebuilding fails."
msgstr ""
"Esto puede ser necesario cuando una de las anotaciones es un ForwardRef que "
"no pudo ser resuelto durante el intento inicial de construir el esquema, y "
"la reconstrucción automática falla."

#: f78393eb782d477d89f3c6507173ab5b of pydantic.main.BaseModel.model_rebuild:7
msgid "Args:"
msgstr "Args:"

#: ffa90021ed8042f69c5a6e69b4afc68d of pydantic.main.BaseModel.model_rebuild:8
msgid ""
"force: Whether to force the rebuilding of the model schema, defaults to "
"`False`. raise_errors: Whether to raise errors, defaults to `True`. "
"_parent_namespace_depth: The depth level of the parent namespace, defaults "
"to 2. _types_namespace: The types namespace, defaults to `None`."
msgstr ""
"force: Si se debe forzar la reconstrucción del esquema del modelo, por "
"defecto es `False`. raise_errors: Si se deben generar errores, por defecto "
"es `True`. _parent_namespace_depth: El nivel de profundidad del espacio de "
"nombres padre, por defecto es 2. _types_namespace: El espacio de nombres de "
"tipos, por defecto es `None`."

#: b3b028164d164355aa1282dbbe378f3e of
#: pydantic.main.BaseModel.model_rebuild:13
msgid "Returns:"
msgstr "Devoluciones:"

#: 2d346f1bc44942ef91a81ba4ab29295d of
#: pydantic.main.BaseModel.model_rebuild:14
msgid ""
"Returns `None` if the schema is already \"complete\" and rebuilding was not "
"required. If rebuilding _was_ required, returns `True` if rebuilding was "
"successful, otherwise `False`."
msgstr ""
"Devuelve `None` si el esquema ya está \"completo\" y no se requería "
"reconstrucción. Si se requería reconstrucción, devuelve `True` si la "
"reconstrucción fue exitosa, de lo contrario, devuelve `False`."
